eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -4,6 +4,56 @@ parameters:
   TestMarkArgument: ''
   BuildTargetingString: 'azure-*'
   TestTimeoutInMinutes: 0
+  TestMatrix:
+    Linux_Python27:
+      OSName: 'Linux'
+      OSVmImage: 'ubuntu-16.04'
+      PythonVersion: '2.7'
+      CoverageArg: ''
+      RunForPR: true
+    Linux_Python35:
+      OSName: 'Linux'
+      OSVmImage: 'ubuntu-16.04'
+      PythonVersion: '3.5'
+      CoverageArg: ''
+      RunForPR: true
+    Linux_Python36:
+      OSName: 'Linux'
+      OSVmImage: 'ubuntu-16.04'
+      PythonVersion: '3.6'
+      CoverageArg: ''
+      RunForPR: false
```

Comment:
Not that it matters too much but can we get away with out listing the CoverageARg and RunForPr = false entries? It would help cut down on the size of this matrix table.